src/python/grpcio/tests/_runner.py

```diff
@@ -105,6 +105,8 @@ def write_bypass(self, value):
     Arguments:
       value (str): What to write to the original file.
     """
+    if sys.version_info.major >= 3 and not isinstance(value, bytes):
```

Comment:
This is logic that I'd expect `six` to afford as a function. Does it not? Or is there some reason `six` is not available to this code?